src/com/google/enterprise/adaptor/database/UniqueKey.java

```diff
@@ -119,23 +129,23 @@
     if ("".equals(contentSqlColumns.trim())) {
       contentSqlCols = names;
     } else {
-      contentSqlCols = splitIntoNameList(contentSqlColumns, tmpTypes.keySet());
+      contentSqlCols = splitIntoNameList(contentSqlColumns);
     }
 
     if ("".equals(aclSqlColumns.trim())) {
       aclSqlCols = names;
     } else {
-      aclSqlCols = splitIntoNameList(aclSqlColumns, tmpTypes.keySet());
+      aclSqlCols = splitIntoNameList(aclSqlColumns);
     }
   }
 
-  private static List<String> splitIntoNameList(String cols,
-      Set<String> validNames) {
+  private List<String> splitIntoNameList(String cols) {
```

Comment:
Now I would certainly undo the s/static// and s/types.get/getType/ changes below.